src/govuk/helpers/_links.scss

```diff
@@ -10,18 +10,48 @@
 
 @mixin govuk-link-common {
   @include govuk-typography-common;
+  @include govuk-link-decoration;
+
+  &:hover {
+    @include govuk-link-hover-decoration;
```

Comment:
This separation is just to support details, which may be changing soon anyway – the alternative would be to duplicate more of the CSS in the details component? But this might be a helpful separation for anyone building their own components where the hover interaction is a little more complex.